app/src/components/map/MapContainer.tsx

```diff
@@ -87,6 +103,32 @@ const MapContainer: React.FC<IMapContainerProps> = (props) => {
         <LayersControl.BaseLayer name="BC Government">
           <TileLayer url="https://maps.gov.bc.ca/arcgis/rest/services/province/roads_wm/MapServer/tile/{z}/{y}/{x}" />
         </LayersControl.BaseLayer>
+        {hideOverlayLayers !== true && (
+          <>
+            <LayersControl.Overlay name="Wildlife Management Units">
+              <WFSFeatureGroup
+                name={'Wildlife Management Units'}
+                typeName={'pub:WHSE_WILDLIFE_MANAGEMENT.WAA_WILDLIFE_MGMT_UNITS_SVW'}
+                minZoom={8}
```

Comment:
question: why is minzoom here different than other predefined geometries?

Reply:
Arbitrarily chosen by me, but its based on how large the features are, and how many there are.

If the features are large, and there are fewer, then a lower zoom level is fine.  But some layers have a huge number of small features, so a higher zoom level would be better.